curvefs/src/mds/fs_manager.cpp

```diff
@@ -619,5 +621,114 @@ FSStatusCode FsManager::ReloadMountedFsVolumeSpace() {
     return FSStatusCode::OK;
 }
 
+FSStatusCode FsManager::IncreaseFsTxSequence(const std::string& fsName,
```

Comment:
I think it's better to extract rename-related things into one place because some of them are in topology, while some others are in fsinfo.

Reply:
> I think it's better to extract rename-related things into one place because some of them are in topology, while some others are in fsinfo.

i think it's inevitable, because the `txId` is belongs to partition, so we will store it in topology, but the `txSequence` is belongs to filesystem which is globally incremented, so we must store it in `fsinfo`.